packages/http/src/mocker/generator/__tests__/JSONSchema.spec.ts

```diff
@@ -9,15 +9,37 @@ describe('JSONSchema generator', () => {
         properties: {
           name: { type: 'string', minLength: 1 },
           email: { type: 'string', format: 'email' },
+          ip: { type: 'string', 'x-faker': 'internet.ip' },
         },
-        required: ['name', 'email'],
+        required: ['name', 'email', 'ip'],
       };
 
       const instance = generate(schema);
       expect(instance).toHaveProperty('name');
       expect(instance).toHaveProperty('email');
     });
 
+    it('generates dynamic example with faker from schema', () => {
+      const schema: JSONSchema = {
+        type: 'object',
+        properties: {
+          name: { type: 'string', minLength: 1 },
+          email: { type: 'string', format: 'email' },
+          ip: { type: 'string', 'x-faker': 'internet.ip' },
+        },
+        required: ['name', 'email', 'ip'],
+      };
+
+      const instance = generate(schema);
+      expect(instance).toHaveProperty('name');
+      expect(instance).toHaveProperty('email');
+      expect(instance).toHaveProperty('ip');
+      // @ts-ignore
+      expect(instance.ip).not.toBeUndefined();
```

Comment:
I would say the first expect is not required — the previous one (`https://github.com/stoplightio/prism/pull/384/files#diff-8e6fad9f7c42e5f1bdb26a87b73283b0R36`) is implying the second one

Reply:
950e6347233b712d6d74d36c3c86b4537102cece